drive.go

```diff
@@ -165,3 +160,13 @@ func (ds *DriveService) Search(ctx context.Context, q string, filter string) (*I
 
 	return items, resp, nil
 }
+
+func (ds *DriveService) pathEscapeIncludingQuotes(query string) string {
+	// We need to escape single quotes with double single
+	// quotes, otherwise API will fail with unterminated
+	if strings.Contains(query, "'") {
```

Comment:
I don't think the `if` is needed, you can just call `ReplaceAll` and it'll do that check internally I'd expect